### ISSUE TITLE
Create PCE App for Blindnet Privacy & Talent Portal

### ISSUE THREAD

**Issue by milstan** [blindnet-io/dogfood#5]
We need to register an App on DevKit for our own dogfood.

This App needs to serve:
- the Privacy Portal, where users can make requests
- the Talent Portal, that can send consents, and events (data capture event, once we support them), and have a connected storage
- Any other storage we may want to connect in the future (such as MailChimp for our practices of keeping people informed, and Apollo used by salesteam, Silae for employee data)

Therefore, we need it configured with the following settings:
### general-info
```
{
  "countries": [
    "France",
    "USA"
  ],
  "organization": "blindnet Inc.",
  "dpo": "Vuk Janosevic, **<INPUT HERE THE ADRESS OF PRIVACY_PORTAL>**",
  "data_consumer_categories": [
    "Blindnet employees",
    "Blindnet's DPO",
    "Blindnet's Legal, Accounting and FInancial Services providers under NDA"
  ],
  "privacy_policy_link": "https://www.blindnet.io/legal/privacy-policy",
  "data_security_info": "We use administrative, technical, and physical safeguards to protect your personal data, taking into account the nature of the personal data and the processing, and the threats posed."
}
```
### legal-bases
```
 [ {"lb_type": "CONSENT",
  "name": "Collection of data through Job Application Form",
  "description": "Collection of the contact data, user identifiers, and CVs for hiring purposes",
  "scope": [
    {
      "dc": ["AFFILIATION.MEMBERSHIP", "AFFILIATION.SCHOOL", "AFFILIATION.WORKPLACE", "DEMOGRAPHIC.AGE", "CONTACT", "IMAGE", "UID", "OTHER-DATA"],
      "pc": ["COLLECTION", "STORING", "USING"],
      "pp": "EMPLOYMENT"
    }
  ]
},

 {"lb_type": "CONTRACT",
  "name": "Silae storage of employee information",
  "description": "Collection and keeping of the contact, CVs and ID of current employees for issuing payslips and compliance with employment regulations",
  "scope": [
    {
      "dc": ["AFFILIATION.MEMBERSHIP", "AFFILIATION.SCHOOL", "AFFILIATION.WORKPLACE", "DEMOGRAPHIC.AGE", "CONTACT", "IMAGE", "UID", "OTHER-DATA"],
      "pc": ["COLLECTION", "STORING", "USING"],
      "pp": ["EMPLOYMENT", "COMPLIANCE"]
    }
  ]
},

 {"lb_type": "NECESSARY.LEGAL-OBLIGATION",
  "name": "Silae storage of employee information",
  "description": "Keeping of the contact, CVs and IDs of current and past employees for purposes of accountability with regards to public administration",
  "scope": [
    {
      "dc": ["AFFILIATION.MEMBERSHIP", "AFFILIATION.SCHOOL", "AFFILIATION.WORKPLACE", "DEMOGRAPHIC.AGE", "CONTACT", "IMAGE", "UID", "OTHER-DATA"],
      "pc": ["COLLECTION", "STORING", "USING"],
      "pp": ["EMPLOYMENT", "COMPLIANCE"]
    }
  ]
},

 {"lb_type": "LEGITIMATE-INTEREST",
  "name": "Using contact data to propose potential collaboration",
  "description": "Collecting, keeping and using contact data for purposes of exploring the possibility of collaboration",
  "scope": [
    {
      "dc": ["AFFILIATION.WORKPLACE", "CONTACT", "UID"],
      "pc": ["COLLECTION", "STORING", "USING"],
      "pp": ["ADVERTISING", "MARKETING", "SERVICES"]
    }
  ]
},

{"lb_type": "CONSENT",
  "name": "Using contact data to send business updates to investors and followers through MailChimp",
  "description": "Collecting, keeping and using contact data in MailChimp for purposes of sending business updates",
  "scope": [
    {
      "dc": ["AFFILIATION.WORKPLACE", "CONTACT", "UID"],
      "pc": ["COLLECTION", "STORING", "USING"],
      "pp": ["MARKETING", "OTHER-PURPOSE"]
    }
  ]
}

]
```
### retention-policies
```
[
  {
    "data_category": ["AFFILIATION",  "DEMOGRAPHIC", "CONTACT", "IMAGE", "UID", "OTHER-DATA"],
    "policy": "NO-LESS-THAN",
    "duration": "P5Y",
    "after": "SERVICE-END"
  }
]
```
### provenances
```
[
  {
    "data_category": ["AFFILIATION", "CONTACT", "DEMOGRAPHIC", "IMAGE", "UID", "OTHER-DATA"],
    "provenance": "USER.DATA-SUBJECT",
    "system": "https://blindnet.io"
  },
  {
    "data_category": ["AFFILIATION", "CONTACT", "UID"],
    "provenance": ["DERIVED", "TRANSFERRED"],
    "system": "https://blindnet.io"
  }
]
```

### Other Considerations

@m4rk055 Does the Endpoint allow to provide Privacy Scope (and Privacy Scope dimension) values as arrays (that PCE is to resolve by generating all possible combinations of Privacy Scope Tripples)? Or otherwise is the configuring user expected to make all the tripples? I made this configuration assuming the first (if not please consider adapting the endpoint to it, as it will be very difficult to have DPOs generate all the tripples.

In other words, wherever there is a Privacy Scope dimension to be provided, assume it can be an array. Then resolve to ass possible tripple combinations of all array values.

**Comment by m4rk055** [comment]
The app is registered and configured.

**Comment by milstan** [comment]
Thanx
